rotkehlchen/tests/api/test_accounting_rules.py

```diff
@@ -523,3 +524,123 @@ def test_cache_invalidation(rotkehlchen_api_server: APIServer):
+            ), json={
+                'directory_path': temp_directory,
+            },
+        )
+        assert_proper_response_with_result(response)
+
+        rules_file_path = Path(temp_directory) / 'accounting_rules.json'
+        assert rules_file_path.exists()
+
+        with open(rules_file_path, encoding='utf-8') as f:
+            rules_data = json.load(f)
+
+        assert len(rules_data['accounting_rules']) == 82
+        assert rules_data['accounting_rules']['1'] == {
+            'event_type': 'deposit',
+            'event_subtype': 'deposit asset',
+            'counterparty': 'aave-v1',
+            'rule': {
+                'taxable': {
+                    'value': False,
+                },
+                'count_entire_amount_spend': {
+                    'value': False,
+                },
+                'count_cost_basis_pnl': {
+                    'value': True,
+                },
+                'accounting_treatment': 'swap',
+            },
+        }
+        assert rules_data['accounting_rules']['82'] == {
+            'event_type': 'deposit',
+            'event_subtype': 'fee',
+            'counterparty': None,
+            'rule': {
+                'taxable': {
+                    'value': True,
+                },
+                'count_entire_amount_spend': {
+                    'value': False,
+                },
+                'count_cost_basis_pnl': {
+                    'value': True,
+                },
+                'accounting_treatment': None,
+            },
+        }
+        assert rules_data['linked_properties'] == {
+            '1': {
+                'accounting_rule': 46,
+                'property_name': 'count_cost_basis_pnl',
+                'setting_name': 'include_crypto2crypto',
+            },
+            '2': {
+                'accounting_rule': 65,
+                'property_name': 'taxable',
+                'setting_name': 'include_gas_costs',
+            },
+            '3': {
+                'accounting_rule': 65,
+                'property_name': 'count_entire_amount_spend',
+                'setting_name': 'include_gas_costs',
+            },
+            '4': {
+                'accounting_rule': 65,
+                'property_name': 'count_cost_basis_pnl',
+                'setting_name': 'include_crypto2crypto',
+            },
+        }
+
+        # edit the rules and setting properties
+        with rotkehlchen_api_server.rest_api.rotkehlchen.data.db.conn.write_ctx() as write_cursor:
+            write_cursor.execute(
+                'UPDATE accounting_rules SET identifier = 83 WHERE identifier = 82;',
+            )
+            write_cursor.execute(
+                'UPDATE linked_rules_properties SET setting_name = "include_gas_costs" WHERE identifier = 2;',  # noqa: E501
+            )
+
+        response = requests.put(
+            api_url_for(  # import the accounting rules from this json file
+                rotkehlchen_api_server,
```

Comment:
can you also check if uploading as multipart works? It is good to have it tested since once happened that it wasn't properly implemented and we didn't notice. Just check that the endpoint response is okey